frontend/src/admin/components/clone-user-access/clone-user-access.tsx

```diff
@@ -0,0 +1,8 @@
+import './clone-user-access.scss'
+
+export default function CloneUserAccess() {
```

Comment:
Is it necessary to have this component?
And if it is needed: The component name is phrased as if it is an action, then perhaps it should be changed to something like `CloneUserAccessButton`.